src/js/base/api.js

```diff
@@ -812,11 +812,11 @@ ripe.Ripe.prototype._getImageOptions = function(options = {}) {
     }
 
     if (options.offsets !== undefined && options.offsets !== null) {
-        params.offsets = options.offsets;
+        params.offsets = JSON.stringify(options.offsets);
     }
 
     if (options.curve !== undefined && options.curve !== null) {
-        params.curve = options.curve;
+        params.curve = JSON.stringify(options.curve);

```

Comment:
How did you test this?

These example URLs are not using JSON for that field:
- https://ripe-core-sbx.platforme.com/test/config/render?brand=kirkwood&model=beya_loafer&initials=AAA&initials_debug=1&curve=[[0.2,%200.2],%20[0.7,%200.2],%20[0.2,%200.5],%20[0.7,%200.5]]&cache=0&crop=1
- https://ripe-core-sbx.platforme.com/compose?brand=kirkwood&model=beya_loafer&initials=AAA&initials_debug=1&curve=[[0.2,%200.2],%20[0.7,%200.2],%20[0.2,%200.5],%20[0.7,%200.5]]&cache=0&crop=1

Reply:
The generated URL wasn't working, I needed to do this so that the URL is generated correctly

Reply:
Can you show me an example of a compose URL working using JSON-encoding for `curve`? Does it look similar to what I posted?

Reply:
Ok apparently it's correct, just checked